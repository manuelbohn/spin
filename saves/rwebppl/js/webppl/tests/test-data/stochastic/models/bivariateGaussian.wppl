var model = function() {
  // p([x, y]) = N([1, 2], [[1, 0.95], [0.95, 1]]);
  var x = gaussian(1, 1);
  var y = gaussian(1.05 + 0.95 * x, Math.sqrt(1 - Math.pow(0.95, 2)));
  return y;
};
