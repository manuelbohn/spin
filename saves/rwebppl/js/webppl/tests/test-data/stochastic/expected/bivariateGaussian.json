{
  "mean": 2,
  "std": 1
}
